packages/boxel-ui/addon/components/icon-button/index.gts

```diff
@@ -32,19 +32,18 @@ class IconButton extends Component<Signature> {
 
   <template>
     <button
-      class={{cn
-        (if @variant (concat @variant))
-        @class
-      }}
+      class={{cn (if @variant (concat @variant)) @class}}
       {{on 'mouseenter' this.onMouseEnterButton}}
       {{on 'mouseleave' this.onMouseLeaveButton}}
       ...attributes
     >
+      {{! Using inline style attribute because targeting the svg using <style> does not work - css scoping works incorrectly }}
       {{#if @icon}}
         {{svgJar
           @icon
           width=(if @width @width '16px')
           height=(if @height @height '16px')
+          style='margin: auto; display: block;'
```

Comment:
The issue I am fixing here is that the following css from this file:

```
button > svg {
  display: block;
  margin: auto;
}
```

Results in the following scoped CSS, which is not actually targeting what it is is supposed to:

<img width="293" alt="image" src="https://github.com/cardstack/boxel/assets/273660/c8c27251-37f6-48e8-8810-3dabbdfa369c">


@backspace I suppose this a scoped css bug? It looks to me that the scope should be attached to the `button` and not to `svg`